Sources/Kommunicate/Classes/Kommunicate.swift

```diff
@@ -116,6 +116,39 @@ open class Kommunicate: NSObject, Localizable {
         }
     }
 
+    public class func isDeviceJailbroken() -> Bool {
+        // Return false immediately if root detection is disabled in the configuration
+        guard defaultConfiguration.rootDetection else { return false }
+        
+        // Return false if the app is running on a simulator
+        #if targetEnvironment(simulator)
+        return false
+        #endif
+        
+        // List of paths for suspicious apps that indicate jailbreaking
+        let suspiciousAppsPaths = [
+            "/Applications/Cydia.app",
+            "/Applications/blackra1n.app",
+            "/Applications/FakeCarrier.app",
+            "/Applications/Icy.app",
```

Comment:
What this signifies? Are these list of apps used for jailbreaking or something else, will this list changes in future?

Reply:
These are the apps that is installed when the device is Jailbreaked. Referred to the list that available online. Yes we can add apps in this list to restrict those.

Reply:
This list is also shared in VPAT report.